lib/index.js

```diff
@@ -0,0 +1,109 @@
+'use babel';
+
+import path from 'path';
+import { CompositeDisposable } from 'atom';
+
+export default {
+  config: {
+    erbExecutablePath: {
+      description: 'Path to the `erb` executable',
+      type: 'string',
+      default: 'erb'
+    },
+    trimMode: {
+      description: 'What trim mode ERB should use',
+      type: 'string',
+      enum: ['None', '0', '1', '2', '-'],
+      default: 'None'
+    },
+    rubyExecutablePath: {
+      description: 'Path to the `ruby` executable',
+      type: 'string',
+      default: 'ruby'
+    }
+  },
+
+  activate() {
+    require('atom-package-deps').install();
+    this.subscriptions = new CompositeDisposable();
+
+    this.subscriptions.add(
+      atom.config.observe('linter-erb.erbExecutablePath', erbExecutablePath => {
+        this.erbCommand = erbExecutablePath;
+      })
+    );
+    this.subscriptions.add(
+      atom.config.observe('linter-erb.trimMode', trimMode => {
+        this.trimMode = trimMode;
+      })
+    );
+    this.subscriptions.add(
+      atom.config.observe('linter-erb.rubyExecutablePath', rubyExecutablePath => {
+        this.rubyCommand = rubyExecutablePath;
+      })
+    );
+  },
+
+  deactivate() {
+    this.subscriptions.dispose();
+  },
+
+  provideLinter() {
+    return {
+      name: 'ERB',
+      grammarScopes: ['text.html.erb', 'text.html.ruby'],
+      scope: 'file',
+      lintOnFly: true,
+      lint: textEditor => {
+        const helpers = require('atom-linter');
```

Comment:
Import helpers in `provideLinter`, it's not a good idea to require on each lint call